ames_qsar/gethlenergy.py

```diff
@@ -41,5 +40,7 @@ def gethlenergy(dir_path, filename):
 
             prevLine = line
         #end of the searchfile loop
+        if term_flag ==0:

```

Comment:
term_flag => normal_termination or terminated_normally:
terminated_normally = True
if not terminated_normally: